packages/date/.swcrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
❓ could the .swcrc config also be shared?
 having one base config and then import/export it here?

Reply:
Same thing came up in @andipaetzold 's PR there isn't really a nice way today to share configs in SWC so we are leaving them up to the packages themselves

Reply:
i've done the same now